### PR TITLE
loki() destination

### PR DESCRIPTION
**Pull request by zsoltgyulai94**
Documented Loki Grafana destination.
Included new reusable option snippets.

### CODE REVIEW

File: _includes/doc/admin-guide/options/channel-args.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## channel-args()
+
+|  Type:|      |
```
Comment (HofiOne) [inline _includes/doc/admin-guide/options/channel-args.md:3]: could you plz format these tables with spaces to see the table structure (equal-width columns, fields, here, and everywhere bellow as well)

File: _includes/doc/admin-guide/options/channel-args.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+|Default:| -|
+
+*Description:* The `channel-args()` option is available in gRPC-based drivers. The option accepts name-value pairs and sets channel arguments defined in the [GRPC Core library documentation](https://grpc.github.io/grpc/core/group__grpc__arg__keys.html).
```
Comment (HofiOne) [inline _includes/doc/admin-guide/options/channel-args.md:6]: use the external_links.yml for external links plz (here, and everywhere bellow as well)

File: doc/_admin-guide/070_Destinations/125_Loki/001_Loki_options.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+title: loki() destination options
+id: adm-dest-loki-opt
+---
+
+The `loki()` driver sends messages to a Loki Grafna database and has the following options:
```
Comment (HofiOne) [inline doc/_admin-guide/070_Destinations/125_Loki/001_Loki_options.md:6]: small typo `Grafna`

**Review by HofiOne — CHANGES_REQUESTED** [review]

File: _includes/doc/admin-guide/options/channel-args.md (hunk 1 above)
Comment (zsoltgyulai94) [reply _includes/doc/admin-guide/options/channel-args.md]: fixed

File: doc/_admin-guide/070_Destinations/125_Loki/001_Loki_options.md (hunk 3 above)
Comment (zsoltgyulai94) [reply doc/_admin-guide/070_Destinations/125_Loki/001_Loki_options.md]: fixed all typos

File: doc/_admin-guide/070_Destinations/125_Loki/001_Loki_options.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+```config
+labels(
+    "label-output-name" => "field-of-the-message"
+)
+```
+
+## template()
+
+|  Type:|     template or template-function|
+|Default:| $ISODATE $HOST $MSGHDR$MSG|
```
[the remark this answers is not in the document]
Comment (zsoltgyulai94) [reply doc/_admin-guide/070_Destinations/125_Loki/001_Loki_options.md]: fixed

**Review by HofiOne — APPROVED** [review]

**Pull request merged**